stubs/protobuf/METADATA.toml

```diff
@@ -1,2 +1,2 @@
-version = "3.20.*"
+version = "4.21.*"
```

Comment:
```suggestion
version = "3.21.*"
```

Sort of a holdover in the meantime

Reply:
(if you like the idea, feel free to commit it and merge - otherwise we can just wait.)

Reply:
Sure, let's do it!